pkg/resource/table/hooks.go

```diff
@@ -161,7 +213,7 @@ func (rm *resourceManager) syncTableTags(
 ) (err error) {
```

Comment:
and potentially would be good to correct the misordering of the above `desired` and `latest` parameters please.